### PR TITLE
Add instructions for pip installing the latest wheel.

### PR DESCRIPTION
**Pull request by robertnishihara**
We should not merge this until #1597 is fixed.

cc @richardliaw

### CODE REVIEW

**Comment by richardliaw** [comment]
We should also to make sure the landing page of docs and readme of the repo are somewhat consistent

**Comment by robertnishihara** [comment]
Makes sense. Are you referring to the code example? I don't think we need to put the nightly pip installs on the doc landing page.

**Comment by robertnishihara** [comment]
@richardliaw this is certainly visually cleaner, but people won't know what to do with the wheels. We need to tell them to do `pip install -U $WHEEL_NAME` so I think we should just include the full command.

How about we make a separate documentation page for "Trying the Latest Version" and move it off of the README?

**Comment by robertnishihara** [comment]
#1597 has been fixed so there is nothing blocking this PR anymore.

**Comment by robertnishihara** [comment]
Merging as is, but we can always improve later.

**Comment by robertnishihara** [comment]
Then you should also remove the documentation for installing on Mac, right?

**Review by richardliaw — APPROVED** [review]
this is fine with me but someone else should also review, as I basically took over this PR..

File: README.rst [hunk 1]
```diff
@@ -19,40 +43,14 @@ Ray comes with libraries that accelerate deep learning and reinforcement learnin
...
 
-
 Installation
 ------------
 
-- Ray can be installed on Linux and Mac with ``pip install ray``.
-- To build Ray from source, see the instructions for `Ubuntu`_ and `Mac`_.
-
-.. _`Ubuntu`: http://ray.readthedocs.io/en/latest/install-on-ubuntu.html
-.. _`Mac`: http://ray.readthedocs.io/en/latest/install-on-macosx.html
```
Comment (robertnishihara) [inline README.rst]: Shouldn't we remove this documentation page?

Comment (richardliaw) [reply README.rst]: hm is it not removed?

Comment (robertnishihara) [reply README.rst]: Oh my mistake.

**Comment by robertnishihara** [comment]
Looks good to me, I'd suggest merging.

**Comment by richardliaw** [comment]
Test failures unrelated - lint passes - merging

**Pull request merged**